db/migrate/20151221234035_add_is_active_to_chapters.rb

```diff
@@ -0,0 +1,5 @@
+class AddIsActiveToChapters < ActiveRecord::Migration
+  def change
+    add_column :chapters, :is_active, :integer, :default => 1
```

Comment:
If this is meant to represent true or false (which the column name implies), I would go ahead and make this a boolean.